docs/getting-started/configuration.md

```diff
@@ -0,0 +1,311 @@
+
+<p align='center'>
+  <img src='../assets/getting-started/gh-oauth.png' alt='Screenshot of the GitHub OAuth creation page' />
+</p>
+
+Set the scope to your likings. For this tutorial, selecting "repo" should be
+enough.
+
+In the `app-config.yaml`, search for `integrations:` and add your token, like we
+did in below example:
+
+```
+integrations:
+  github:
+    - host: github.com
+      token: ghp_urtokendeinfewinfiwebfweb
+```
+
+That's settled. This information will be leveraged by other plugins.
+
+Some helpful links, for if you want to learn more about:
+
+- [Other available integrations](../integrations/)
+- [Using GitHub Apps instead of a Personal Access Token](../plugins/github-apps#docsNav)
+
+### Explore what we've done so far
+
+## Login to Backstage and check profile
+
+Open your Backstage frontend. You should see your login screen if you're not
+logged in yet. As soon as you've logged in, go to Settings, you'll see your
+profile. Hopefully you'll recognize the profile picture and name on your screen,
+otherwise something went terribly wrong.
+
+## Register an existing component
+
+- Register a new component, by going to `create` and choose
+  `Register existing component`
+
+<p align='center'>
+  <img data-zoomable src='../assets/getting-started/b-existing-1.png' alt='Software template main screen, with a blue button to add an existing component' />
+</p>
+
+- As URL use `https://github.com/backstage/demo/blob/master/catalog-info.yaml`.
+  This is used by our [demo site](https://demo.backstage.io).
+
+<p align='center'>
+  <img src='../assets/getting-started/b-existing-2.png' alt='Register a new component wizard, asking for an URL to the existing component YAML file' />
+</p>
+- Hit `Analyze` and review the changes. Apply them if correct
+
+<p align='center'>
+  <img src='../assets/getting-started/b-existing-3.png' alt='Register a new component wizard, showing the metadata for the component YAML we use in this tutorial' />
+</p>
+
+- You should receive a message that your entities have been added.
+- If you go back to `Home`, you should be able to find `demo`. You should be
+  able to click it and see the details
+
+## Create a new component using a software template
+
+- Go to `create` and choose to create a website with the `React SSR Template`
+- Type in a name, let's use `tutorial`
+- Select the group `group-a` which will own this new website, and go to the next
+  step
+
+<p align='center'>
+  <img src='../assets/getting-started/b-scaffold-1.png' alt='Software template deployment input screen asking for a name, the group owning this, and a description' />
+</p>
+
+- For the location, we're going to use the default
+- As owner, type your GitHub username
+- For the repository name, type `tutorial`. Go to the next step
+
+<p align='center'>
+  <img src='../assets/getting-started/b-scaffold-2.png' alt='Software template deployment input screen asking for the github username, and name of the new repo to create' />
```

Comment:
```suggestion
  <img src='../assets/getting-started/b-scaffold-2.png' alt='Software template deployment input screen asking for the GitHub username, and name of the new repo to create' />
```

this may fix your markdown spell check error in the pipeline